src/core/TransformControls.tsx

```diff
@@ -17,20 +17,19 @@ declare global {
 
 type Props = TransformControls &
   JSX.IntrinsicElements['group'] & {
-    enabled: boolean
-    axis: string | null
-    mode: string
-    translationSnap: number | null
-    rotationSnap: number | null
+    enabled?: boolean
+    axis?: string | null
+    mode?: string
+    translationSnap?: number | null
+    rotationSnap?: number | null
     scaleSnap?: number | null
-    space: string
-    size: number
-    dragging: boolean
```

Comment:
Removed this prop as it is a readonly for TransformControls.
https://threejs.org/docs/index.html?q=trans#examples/en/controls/TransformControls
.dragging : Boolean
Whether or not dragging is currently performed. Read-only property.